modules/miner/miner.go

```diff
@@ -16,9 +16,15 @@ const (
 
 	// headerForWorkMemory is the number of previous calls to 'headerForWork'
 	// that are remembered.
-	headerForWorkMemory = 50
+	headerForWorkMemory = 20
```

Comment:
I think I'm going to change the constants to `headerForWorkMemory` and `blockForWorkMemory` (max number of blocks to be stored) because I think it will make code more understandable.